### PR TITLE
[4.2] codemirror full screen

### PR DESCRIPTION
**Pull request by brianteeman**
In com_templates if you display the original file as well as the override then you have two instances of codemirror with the original file as readonly. Both have the message "Press xx to toggle Full Screen editing"  which doesnt make sense on a readonly instance of the editor and not surprisingly doesnt work.

This PR removes the message and associated options from the codemirror layout if the editor is defined as readonly in the xml

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

### CODE REVIEW

File: plugins/editors/codemirror/layouts/editors/codemirror/element.php [hunk 1]
```diff
@@ -37,6 +37,11 @@
 $editor          = 'editor="' . ltrim(HTMLHelper::_('script', $basePath . 'lib/codemirror' . $extJS, ['version' => 'auto', 'pathOnly' => true]), '/') . '?' . $mediaVersion . '"';
 $addons          = 'addons="' . ltrim(HTMLHelper::_('script', $basePath . 'lib/addons' . $extJS, ['version' => 'auto', 'pathOnly' => true]), '/') . '?' . $mediaVersion . '"';
 
+// remove the fullscreen message and option if readonly not null.
```
Comment (sandewt) [inline plugins/editors/codemirror/layouts/editors/codemirror/element.php]: `// remove ...`

should be: (uppercase)

`// Remove ...`

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on c944a91be7d11f74d869da2610c6145d4de4b532<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38863">issues.joomla.org/tracker/joomla-cms/38863</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on c944a91be7d11f74d869da2610c6145d4de4b532<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38863">issues.joomla.org/tracker/joomla-cms/38863</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38863">issues.joomla.org/tracker/joomla-cms/38863</a>.</sub>

**Comment by roland-d** [comment]
Thank you

**Pull request merged**